alignak_backend/template.py

```diff
@@ -405,6 +405,18 @@ def fill_template_host(item):
                     if key not in item:
                         item['_template_fields'][key] = 0
 
+        if 'check_command' not in item:
+            # Get default host check commands
+            commands = current_app.data.driver.db['command']
+            default_host_check_command = commands.find_one({'name': '_internal_host_up'})
+            item['check_command'] = default_host_check_command['_id']
+
+        if '_realm' not in item:
+            # Get default realm
+            realms = current_app.data.driver.db['realm']
+            default_realm = realms.find_one({'name': 'All'})
+            item['_realm'] = default_realm['_id']
+
```

Comment:
I agree with you... we should use the highest level realm the user is authorized to post in...